ui/app/pages/home/index.scss

```diff
@@ -36,4 +36,61 @@
   &__tab {
     flex-grow: 1;
   }
+
+  &__connect-status-text {
+    display: flex;
+    flex-direction: column;
+    @extend %content-text;
+    padding-left: 24px;
+    padding-right: 24px;
+
+    div {
+      &:last-child {
+        margin-top: 26px;
+      }
+    }
+  }
+
+  &__connected-status-popover {
+    width: 329px;
+    height: 295px;
+    margin-top: -12px;
+
+    .popover-header {
+      padding-bottom: 4px;
+    }
+
+    .popover-content {
+      overflow-y: auto;
+    }
+  }
+
+  &__connected-status-popover-bg {
+    height: 34px;
+    width: 110px;
+    border-radius: 34px;
+    position: absolute;
+    top: 82px;
+    left: 12px;
+    opacity: 1;
+    box-shadow: 0 0 0 9999px rgba(0, 0, 0, 0.2);
+    background: none;
+  }
+
+  &__connected-status-popover-bg-container {
+    height: 100%;
+    width: 100%;
+  }
+
+  &__connect-status-popover-footer {
```

Comment:
This class isn't being applied - it should say `connected` here, not `connect`.

Reply:
Most of these styles aren't required though. I managed to get it looking reasonable with just three styles: `justify-content: flex-end` on the footer, `width: 133px` on the button, and `margin: 0` on the _button_ (not the footer itself).

Reply:
addressed in 9bfe721f8